doc/source/tutorial/tutorial_indexing.ipyml

```diff
@@ -0,0 +1,466 @@
+    pop[2010:2016, 'BruCap', 50:52, 'F', 'BE']
+
+  id: 3
+
+- code: |
+    # slices bounds are optional:
+    # if not given start is assumed to be the first label and stop is the last one.
+    # Here we select all years starting from 2010
+    pop[2010:, 'BruCap', 50:52, 'F', 'BE']
+
+  id: 4
+
+- code: |
+    # Slices can also have a step (defaults to 1), to take every Nth labels
+    # Here we select all even years starting from 2010
+    pop[2010::2, 'BruCap', 50:52, 'F', 'BE']
+
+  id: 5
+
+- code: |
+    # one can also use list of labels to take non-contiguous labels.
+    # Here we select years 2008, 2010, 2013 and 2015
+    pop[[2008, 2010, 2013, 2015], 'BruCap', 50:52, 'F', 'BE']
+
+  id: 6
+
+- markdown: |
+    The order of indexing does not matter either, so you usually do not care/have to remember about axes positions during computation. It only matters for output.
+
+
+- code: |
+    # order of index doesn't matter
+    pop['F', 'BE', 'BruCap', [2008, 2010, 2013, 2015], 50:52]
+
+  id: 7
+
+- markdown: |
+    <div class="alert alert-warning">
+    **Warning:** Selecting by labels as above works well as long as there is no ambiguity.
+       When two or more axes have common labels, it may lead to a crash.
+       The solution is then to precise to which axis belong the labels.
+    </div>
+
+
+- code: |
+    # let us now create an array with the same labels on several axes
+    age, weight, size = Axis('age=0..80'), Axis('weight=0..120'), Axis('size=0..200')
+    
+    arr_ws = ndtest([age, weight, size])
+
+  id: 8
+
+- code: |
+    # let's try to select teenagers with size between 1 m 60 and 1 m 65 and weight > 80 kg.
+    # In this case the subset is ambiguous and this results in an error:
+    arr_ws[10:18, :80, 160:165]
+
+  id: 9
+
+- code: |
+    # the solution is simple. You need to precise the axes on which you make a selection
+    arr_ws[age[10:18], weight[:80], size[160:165]]
+
+  id: 10
+
+- markdown: |
+    ### Ambiguous Cases - Specifying Axes Using The Special Variable X
+    
+    When selecting, assiging or using aggregate functions, an axis can be
+    refered via the special variable ``X``:
+    
+    -  pop[X.age[:20]]
+    -  pop.sum(X.age)
+    
+    This gives you acces to axes of the array you are manipulating. The main
+    drawback of using ``X`` is that you lose the autocompletion available from
+    many editors. It only works with non-anonymous axes for which names do not contain whitespaces or special characters.
+
+
+- code: |
+    # the previous example could have been also written as
+    arr_ws[X.age[10:18], X.weight[:80], X.size[160:165]]
+
+  id: 11
+
+- markdown: |
+    ### Selecting by Indices
+    
+    Sometimes it is more practical to use indices (positions) along the axis, instead of labels.
+    You need to add the character ``i`` before the brackets: ``.i[indices]``.
+    As for selection with labels, you can use single index or slice or list of indices.
```

Comment:
either "use a single index, a slice or a list of indices" or "use single indices, slices or lists of indices"